packages/peregrine/lib/talons/WishlistPage/__tests__/__snapshots__/useWishlist.spec.js.snap

```diff
@@ -1,9 +1,137 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`it handles editing a wishlist handleEditWishlist() runs the mutation with a thrown error 1`] = `
+Array [
+  [Error: Wish list "Favorites" already exists.],
+]
+`;
+
+exports[`it handles editing a wishlist handleEditWishlist() runs the mutation with the expected values 1`] = `
+Object {
+  "awaitRefetchQueries": true,
+  "refetchQueries": Array [
```

Comment:
I ran into something similar in a snapshot recently, and wasn't a fan of including the entire GraphQLDocument AST in the snap. It would be possible to mock here since this test doesn't make use of the `MockedProvider`. This may be personal preference so consider this suggestion optional.

```js
jest.mock('../wishlist.gql', () => ({
    getCustomerWishlistQuery: jest.fn().mockName('getCustomerWishlistQuery'), 
    updateWishlistMutation: jest.fn().mockName('updateWishlistMutation')
}));
```